Cargo.toml

```diff
@@ -133,6 +133,7 @@ bytemuck = "1.7"
 # Needed to poll Task examples
 futures-lite = "1.11.3"
 crossbeam-channel = "0.5.0"
+wgpu = "*"
```

Comment:
It would be nice to not need to add `wgpu` as a dependency here.
Could you check if it's possible to add what you need to import from `wgpu` to https://github.com/bevyengine/bevy/blob/main/crates/bevy_render/src/render_resource/mod.rs#L26?

Reply:
Done.